vignettes/assertr.Rmd

```diff
@@ -1,16 +1,16 @@
 ---
-title: "Assertive R Programming with assertr"
+  title: "Assertive R Programming with assertr"
```

Comment:
Your editor introduced a lot of whitespace changes in this file. This makes it hard to review. Also we shouldn't introduce whitespace changes if it is not necessary. Please revert whitespace changes and adjust your editor configuration :)